### ISSUE TITLE
support speakers added as homekit smart speakers?

### ISSUE THREAD

**Issue by chris-chensy** [ikalchev/HAP-python#275]
Currently, with home assistant, the speakers (media_player) can only be added as a group of switches in homekit. I wonder if the speakers can be treated as smart speakers so the apple tv can cast Apple Music to the speaker.

**Comment by breitburg** [comment]
Yep. That would be awesome. I found AirPlay 2 receiver implementation on Python [here](https://github.com/Argyropus/airplay2-receiver). I think it's possible to implement that as different class.

**Comment by made2591** [comment]
Yes! I'm looking for the same functionality! I will have a look at the link you posted @breitburg but we should definitely upvote this issue cause I read in the homeassistant forum is a *wanted-feature* as well
